prow/plugins/lgtm/lgtm.go

```diff
@@ -60,9 +60,36 @@ func init() {
 }
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
-	// The Config field is omitted because this plugin is not configurable.
+	configInfo := map[string]string{}
+	for _, orgRepo := range enabledRepos {
+		parts := strings.Split(orgRepo, "/")
+		if len(parts) > 2 {
+			return nil, fmt.Errorf("invalid repo in enabledRepos: %q", orgRepo)
+		}
+		opts := optionsForRepo(config, parts[0], parts[1])
```

Comment:
There may not be a part[1]

Reply:
Damn...

Reply:
Done, would it make sense to actually unittest helpProvider() in lgtm_test?

Reply:
I'll never say no to a unit test :)